validation-status.md

```diff
@@ -0,0 +1,565 @@
+* Remark: A discussion is ongoing if the reference to zib NursingIntervention needs to modelled, as this is deemed unnecessary on the zib level.
+
+### AbilityToPerformMouthcareActivities v3.1
+* Status: not submitted for validation
+
+### AbilityToPerformNursingActivities v1.0.1
+* Status: not submitted for validation
+
+### AbilityToUseToilet v3.1.1
+* Profiles:
+    * zib-AbilityToUseToilet
+    * zib-AbilityToUseToilet.MenstrualCare
+    * zib-AbilityToUseToilet.ToiletUse
+* Status: validated
+
+### AbilityToWashOneself v3.1.1
+* Profile: zib-AbilityToWashOneself
+* Status: validated
+
+### AddressInformation v1.1
+* Profile: zib-AddressInformation
+* Status: validated
+
+### AdministrationAgreement v1.0.3
+* Status: not submitted for validation
+* Remark: For the medication related zibs, there's an ongoing discussion if the profiles will be published as the zibs (and thus the profiles) are incompatible with the requirements of the MP9 program, which is the main use case for exchange of medication information in the Netherlands. Pending this discussion, these profiles are withheld from validation.
+
+### AdvanceDirective v3.1.1
+* Profile: zib-AdvanceDirective
+* Status: in progress
+
+### AlcoholUse v3.2
+* Profile: zib-AlcoholUse
+* Status: validated
+* Remark: Since validation, a new issues has come up regarding the relation between the AlcoholUseStatus and the ObservationOfUse container. This could result in the withdrawal of this profile and subsequent re-profiling of this zib.
+
+### Alert v4.1
+* Profile: zib-Alert
+* Status: in progress
+
+### AllergyIntolerance v3.3
+* Profile: zib-AllergyIntolerance
+* Status: validated
+
+### AnatomicalLocation v1.0
+* Profile: zib-AnatomicalLocation
+* Status: validated
+
+### ApgarScore v1.0.1
+* Profiles:
+    * zib-ApgarScore-1Minute
+    * zib-ApgarScore-5Minute
+    * zib-ApgarScore-10Minute
+* Status: validated
+
+### BarthelADLIndex v3.1
+* Profile: zib-BarthelADLIndex
+* Status: in progress
+
+### BladderFunction v3.2
+* Profiles:
+    * zib-BladderFunction
+    * zib-Stoma
+    * zib-MedicalDevice
+    * zib-MedicalDevice.Product
+* Status: not submitted for validation
+
+### BloodPressure v3.2.1
+* Profile: zib-BloodPressure
+* Status: validated
+
+### BodyHeight v3.1.1
+* Profile: zib-BodyHeight
+* Status: validated
+
+### BodyTemperature v3.1.2
+* Profile: zib-BodyTemperature
+* Status: validated
+
+### BodyWeight v3.2
+* Profile: zib-BodyWeight
+* Status: validated
+
+### BowelFunction v3.1.1
+* Profiles:
+    * zib-BowelFunction
+    * zib-BowelFunction.FecalContinence
+    * zib-BowelFunction.Frequency
+    * zib-BowelFunction.DefecationConsistency
+    * zib-BowelFunction.DefecationColor
+    * zib-MedicalDevice
+* Status: in progress
+
+### Burnwound v3.4
+* Profiles:
+    * zib-Burnwound
+    * zib-Burnwound.Extent
+    * zib-wounds.WoundCharacteristics
+    * zib-wounds.DateOfLastDressingChange
+    * zib-wounds.WoundImage
+* Status: in progress

```

Comment:
Burnwound zi ik niet staan in het HL7 Jira project. Weten we zeker dat deze is aangeleverd?